database/seeds/UserSeeder.php

```diff
@@ -13,13 +13,13 @@ class UserSeeder extends Seeder
     public function run()
     {
         // creating an user
-        $str = 'demopass';
+        $str = array_get($_ENV, 'ADMIN_PASSWORD');
```

Comment:
Isn't `$_ENV['ADMIN_PASSWORD']` working?
Could be also the case to put something like
```php
if (!isset($_ENV['ADMIN_PASSWORD'] || !isset($_ENV['ADMIN_USERNAME']) {
  die('ADMIN_PASSWORD and ADMIN_USERNAME env variables must be set');
}
```
Otherwise simply skipping the operation if those env variables aren't set.

Reply:
I tried with `$_ENV['ADMIN_PASSWORD']` but the variable is not read even if it's set, it only works with the `array_get` function. Maybe could be a good idea to add the `if(!isset)` statement to have more flexibility, and skip the operation if env variables are not set

Reply:
Uhm, that's rather weird. array_get is a helper function from laravel which should be just a wrapper that helps accessing nested maps. If you're really sure it's not working you can try with getenv('ADMIN_PASSWORD'). However, if isset works, also the above method has to work...

Reply:
I'm sorry, I was making a mistake with another syntax, `$_ENV['ADMIN_PASSWORD']` works! I've updated now with this syntax